config/crd/bases/rabbitmq.com_rabbitmqclusters.yaml

```diff
@@ -10,7 +10,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.9.2
+    controller-gen.kubebuilder.io/version: v0.10.0
```

Comment:
Did you mean to bump the kubebuilder version?

Reply:
Not intentionally.
`kubebuilder` seems to come from `controller-gen` which in turn comes from `controller-tools` https://github.com/rabbitmq/cluster-operator/blob/8f4735b5e0f9050bb1028e635c1fba080d57412d/tools/tools.go#L11 which in turn is checked in with `v0.10.0` in https://github.com/rabbitmq/cluster-operator/blob/8f4735b5e0f9050bb1028e635c1fba080d57412d/go.mod#L28

I can revert that line?

Reply:
That was bumped 3 days ago via https://github.com/rabbitmq/cluster-operator/pull/1146
It seems I'm the first one who ran `make manifest` since then. So, it seems correct to bump kubebuilder version as done in this commit?